lib/rubygems/version.rb

```diff
@@ -149,6 +149,10 @@
 # For the last example, single-digit versions are automatically extended with
 # a zero to give a sensible result.
 
+module Gem; end
```

Comment:
Any reason to not define Gem module in "cascade"?

```ruby
module Gem
  class Version
  # ...
```

Reply:
I wanted to keep the diff small instead of having to change the indentation of the whole file. Also, directly opening the class is how other all other files are defined, so it seemed more consistent. That said, I don't mind changing it at all.

Reply:
For me, it would be confusing to just find this definition at the top of the file. At least small comment would make it easier to understand.